integration-tests/flyway/src/test/java/io/quarkus/it/flyway/FlywayFunctionalityTest.java

```diff
@@ -17,4 +18,10 @@
     public void testFlywayQuarkusFunctionality() {
         when().get("/flyway/migrate").then().body(is("1.0.1"));
     }
+
+    @Test
+    @DisplayName("Clean a schema using integrated instance")
+    public void testFlywayCleanQuarkusFunctionality() {
```

Comment:
Not sure what functionality in the extension you're testing here. Doesn't seem related to what the PR proposes.

Reply:
I'm not really sure about this test, tried to do it similar to Migrate test. But testing if the Flyway MigrationInfo is null, That means the history has been undone.

Reply:
Right, it's just that this test seems to test the Flyway API, but you already did that in the other test case. Perhaps this can be removed

Reply:
Done